ros2topic/ros2topic/verb/echo.py

```diff
@@ -64,16 +64,22 @@ def add_arguments(self, parser, cli_name):
             '--truncate-length', '-l', type=unsigned_int, default=DEFAULT_TRUNCATE_LENGTH,
             help='The length to truncate arrays, bytes, and string to '
                  '(default: %d)' % DEFAULT_TRUNCATE_LENGTH)
+        parser.add_argument(
+            '--noarr', action='store_true', help='Exclude arrays')
+        parser.add_argument(
+            '--nostr', action='store_true', help='Exclude string fields')
```

Comment:
Nitpicks:
* for more clarity, should the argument be `--no-str`?
* `help='Don't print string fields of messages.'`

Reply:
Fixed.